src/subscript/casegen_upcars/model.py

```diff
@@ -340,40 +338,32 @@ def __init__(
         self._throws = []
 
         print("Initializing model")
-        print("  LX x LY x LZ: {} x {} x {}".format(self._lx, self._ly, self._lz))
-        print(
-            "  NX x NY x NZ: {} x {} x {}".format(
-                self._total_nx, self._total_ny, self._total_nz
-            )
-        )
-        print("  dx x dy x dz: {} x {} x {}".format(dx, dy, dz))
-
-        print("  Matrix Element in X-direction: {}".format(nMatrixY))
-        print("  Matrix Element in Y-direction: {}".format(nMatrixX))
-        print("  Layers: {}".format(nz))
-        print(
-            "  Shape Factor: {} (a), {} (b), {} (c)".format(
-                radius_x, radius_y, radius_z
-            )
-        )
-        print("  Top geometry: {}".format(top))
-        print("  Model origin: {}, {}".format(origin_x, origin_y))
-        print("  Model coordinate rotation: {}°".format(rotation))
-        print("  Tilting: {}".format(tilt))
+        print(f"  LX x LY x LZ: {self._lx} x {self._ly} x {self._lz}")
+        print(f"  NX x NY x NZ: {self._total_nx} x {self._total_ny} x {self._total_nz}")
+        print(f"  dx x dy x dz: {dx} x {dy} x {dz}")
+
+        print(f"  Matrix Element in X-direction: {nMatrixY}")
+        print(f"  Matrix Element in Y-direction: {nMatrixX}")
+        print(f"  Layers: {nz}")
+        print(f"  Shape Factor: {radius_x} (a), {radius_y} (b), {radius_z} (c)")
+        print(f"  Top geometry: {top}")
+        print(f"  Model origin: {origin_x}, {origin_y}")
+        print(f"  Model coordinate rotation: {rotation}°")
+        print(f"  Tilting: {tilt}")
```

Comment:
Consider using multiline docstrings?
```
import textwrap

x = 44
y = 55


def main():
    multi = f"""
    This is a multiline string
    where things may go wrong for {x}
    but not for {y} and you may
    think
    """

    print(textwrap.dedent(multi))


if __name__ == "__main__":
    main()


"""
```

Reply:
That looked quite cool, but I will leave that for the casegen_upcars developers. :)